pom.xml

```diff
@@ -139,19 +139,21 @@
     <spark2.version>2.4.4</spark2.version>
     <spark3.version>3.5.1</spark3.version>
     <sparkbundle.version></sparkbundle.version>
+    <flink1.20.version>1.19.1</flink1.20.version>
     <flink1.19.version>1.19.1</flink1.19.version>
     <flink1.18.version>1.18.1</flink1.18.version>
     <flink1.17.version>1.17.1</flink1.17.version>
     <flink1.16.version>1.16.2</flink1.16.version>
     <flink1.15.version>1.15.1</flink1.15.version>
     <flink1.14.version>1.14.5</flink1.14.version>
-    <flink.version>${flink1.19.version}</flink.version>
-    <hudi.flink.module>hudi-flink1.19.x</hudi.flink.module>
-    <flink.bundle.version>1.19</flink.bundle.version>
+    <flink.version>${flink1.20.version}</flink.version>
+    <hudi.flink.module>hudi-flink1.20.x</hudi.flink.module>
+    <flink.bundle.version>1.20</flink.bundle.version>
     <!-- This is fixed to match with version from flink-avro -->
     <flink.avro.version>1.11.1</flink.avro.version>
     <flink.format.parquet.version>1.13.1</flink.format.parquet.version>
-    <flink.connector.kafka.version>3.2.0-1.19</flink.connector.kafka.version>
+    <!-- check kafka version -->
```

Comment:
Is this right?

Reply:
I see the discussion regarding flink connector kafka version to 1.20, but it has not been release yet. 
https://github.com/apache/flink-connector-kafka/pull/111

Reply:
got it.